person.rb

```diff
@@ -8,11 +8,16 @@ class Person < Nameable
 
   def initialize(age, name = 'unknown', parent_permission: true)
     super(@nameable = name)
-    @name = name
     @age = age
+    @rentals = []
+    @name = name
     @parent_permission = parent_permission
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] As mentioned [previously](https://github.com/jkanyi-web/OOP_School_Library/pull/3#discussion_r1386453458) the `add_rental` method should pass the person object itself as a parameter when creating the new rental object so I'd suggest you refactor the above method.
- Hint: you can use this code snippet as a reference:
```rb
  def add_rental(book, date)
    # self refers to the person object itself
    Rental.new(date, book, self)
  end
```